src/main/java/com/netchill/services/streaming/StreamingService.java

```diff
@@ -14,26 +14,27 @@ public class StreamingService {
     private MovieDbApiClient movieDbApiClient;
 
     @Inject
-    private StreamingService(MovieDbApiClient movieDbApiClient, MovieDao movieDao){
+    private StreamingService(MovieDbApiClient movieDbApiClient, MovieDao movieDao) {
         this.movieDbApiClient = movieDbApiClient;
     }
 
     public TrailerKey getSerieTrailerById(Long id) {
         Trailer trailer = movieDbApiClient.getSerieTrailerById(id);
         List<TrailerKey> trailerKeys = trailer.getResults();
-        if(trailerKeys.isEmpty()) return new TrailerKey();
-        for(int i=0; i < trailerKeys.size(); i++){
+        if (trailerKeys.isEmpty()) return new TrailerKey();
+        for (int i = 0; i < trailerKeys.size(); i++) {
             if (trailerKeys.get(i).getType().equals("Trailer")) {
                 return trailerKeys.get(i);
             }
         }
         return trailerKeys.get(0);
```

Comment:
à remplacer par un stream.

```suggestion
        return trailerKeys.stream()
            .filter(trailer -> trailer.getType().equals("Trailer"))
            .findFirst()
            .orElseGet(()->  trailerKeys.isEmpty ? new TrailerKey() : trailerKeys.get(0));
```

On pourra en discuter si tu ne comprends pas trop ce que ça fait.
En attendant je t'invite à te documenter sur les streams !